### DISCUSSION TITLE
Paned Browser with Album art ?

### DISCUSSION THREAD

**Discussion by Twix53791** [quodlibet/quodlibet#3823] [Ideas]
Hello,
I would like to be able to choose between displaying album art or not on the paned browser. I think I would be a nice design choice : the goal will be to have a first panel to choose the artist (on a list why not, even it could be great to have also miniatures, but OK), and when you choose an artist, you have the second panel (with the album list) which display the albums arts. Indeed, it is just more beautiful for me, but mostly I it easier to navigate, as we have the habit to identify an album from its cover usually.

Thanks for your consideration,
A nice day

**Comment by EricViala** [comment]
I belatedly but wholeheartedly second this.

**Comment by KarimbaYo** [comment]
Hi there!

I've managed to put together a rather hacky solution by playing around with the source code. _Just a heads-up: I'm not a Python developer, and I relied on AI to get this working, so the code is far from clean or professional._

The display is a bit slower since album covers aren't cached yet, but it's perfectly usable for my needs. It works by adding a pseudo-tag <albumcover> to the column pattern (which I’ve set to multi-line for this purpose).

If you want to test it out, you can find the source here:
[https://github.com/KarimbaYo/quodlibet/tree/feat/paned-browser-covers](https://github.com/KarimbaYo/quodlibet/tree/feat/paned-browser-covers)

<img width="1920" height="1050" alt="Display covers in paned browser" src="https://github.com/user-attachments/assets/fd7cddb2-b3a3-41a4-8490-08be9dd8053e" />

<img width="881" height="761" alt="Preferences in paned browser" src="https://github.com/user-attachments/assets/dd10ea85-8cc6-4fbc-bf17-84b968716195" />
